### PR TITLE
Add UI for scanning ports

### LINKED ISSUE #12: [Frontend] Add GUI with input fields for range + CTAs for scanning network
Currently this is a CLI based application, convert this into a **GUI application** to make the UX better!

- _**Must have:**_ 
     - **A window with menus** (IP Scanning/Port Scanning)
     - **Input box to enter host**
     - **Input fields to enter low & high range of ports**
     - **CTA(Button) to start scanning**
     - **List of Ports/IPs** (once the scanning is done)

- **_Good to have:_**
     - A live timer to show the total scanning time
     - Some kind of indication (animation) in UI while scanning is happening in the background
     - Resume/Pause/Restart scanning options

- **_Extended feature_**
     - UI option to convert a host into its IP address
     - Display history: Last five scans in the following tabular form

|  Date & Time  |     Host     |  IP Address  | Total Scanning time |
|:--------:|:------------:|---|---|
| 9:00 PM June 8, 2020 | localhost | 127.0.0.1  | 24 seconds |

> Recommended to use [PyQt5](https://pypi.org/project/PyQt5/) | Learn more [here](https://build-system.fman.io/pyqt5-tutorial).

**PS: Still open for discussions - feel free to suggest GUI frameworks(preferably from Python).**

### PR DESCRIPTION
**Pull request by ishika1727**
Fixes #12 
Frontend added. Suggested UI changes also done

### CODE REVIEW

File: src/view/js/index.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+$(document).ready(function () {
+    $("#history").show();
+    $("#timer").hide();
+
+
+});
+
+var data = 0;
+function scan_port() {
```
Comment (vinitshahdeo) [inline src/view/js/index.js:9]: use camelCase.

File: src/view/js/index.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    if (data == 0) {
+        data = 1;
+        $("#history").hide();
+        $("#timer").show();
+
+        setInterval(setTime, 1000);
+
+        var minutesLabel = document.getElementById("minutes");
+        var secondsLabel = document.getElementById("seconds");
```
Comment (vinitshahdeo) [inline src/view/js/index.js:18]: @ishika1727 Combine all the `var` declarations.

File: src/view/css/index.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+#colon {
+    font-size: 5em;
+    color: black;
+}
+ i{
+     margin-top: 20vw;
+     
+ }
```
Comment (Kashish121) [inline src/view/css/index.css:90]: `index.css` file needs formatting. CSS will look cleaner. 👀

**Comment by ishika1727** [comment]
Used camelCase and combined variable declarations as suggested by @vinitshahdeo
formatted index.css @Kashish121

**Comment by vinitshahdeo** [comment]
@ishika1727 Awesome 👏

**Comment by vinitshahdeo** [comment]
@Kashish121 Merged the UI changes into `feature/frontend`. Hopefully you're unblocked now.

**Pull request merged**